### DISCUSSION TITLE
Question.  Is it possible to configure the email context of an email via keycloakify?

### DISCUSSION THREAD

**Discussion by Tonypz** [keycloak/keycloak#32374] [Ideas]
Question.  Is it possible to configure the email context of an email via keycloakify?Question.  Is it possible to configure the email context of an email via keycloakify?

**Comment by jonkoops** [comment]
@garronej  FYI

**Comment by garronej** [comment]
Hello @Tonypz,  
Not yet, it's not yet possible to design the emails with Keycloakify.  

If you have question about Keycloakify you should maybe open them on the Keycloakify project's repo
